bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js

```diff
@@ -63,7 +63,9 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
                     title: this.get('discoveryWidgetTitle'),
                     multiple: false,
                     contentDiscoveredHandler: Y.bind(this._setSelectedLocation, this, target),
-                    isSelectable: function () {return true;},
+                    isSelectable: function (contentStruct) {
```

Comment:
What about this line then? https://github.com/ezsystems/content-on-the-fly-prototype-bundle/pull/27/files#diff-2e828c40ca0072a840ecd15ceeccdce1R24